docs/user-guide/station-device-provisioning.md

```diff
@@ -0,0 +1,203 @@
+        ```
+
+      </details>
+
+      <details>
+        <summary>US902 Example Configuration</summary>
+
+        ```json
+        {
+          "routerConfig": {
+            "NetID": [1],
+            "JoinEui": [["DCA632FFFEB32FC5", "DCA632FFFEB32FC7"]],
+            "region": "US902",
+            "hwspec": "sx1301/1",
+            "freq_range": [902000000, 928000000],
+            "DRs": [
+              [10, 125, 0],
+              [9, 125, 0],
+              [8, 125, 0],
+              [7, 125, 0],
+              [8, 500, 0],
+              [0, 0, 0],
+              [0, 0, 0],
+              [0, 0, 0],
+              [12, 500, 1],
+              [11, 500, 1],
+              [10, 500, 1],
+              [9, 500, 1],
+              [8, 500, 1],
+              [8, 500, 1]
+            ],
+            "sx1301_conf": [
+              {
+                "radio_0": { "enable": true, "freq": 902700000 },
+                "radio_1": { "enable": true, "freq": 903400000 },
+                "chan_FSK": { "enable": true, "radio": 1, "if": 300000 },
+                "chan_Lora_std": {
+                  "enable": true,
+                  "radio": 0,
+                  "if": 300000,
+                  "bandwidth": 500000,
+                  "spread_factor": 8
+                },
+                "chan_multiSF_0": { "enable": true, "radio": 0, "if": -400000 },
+                "chan_multiSF_1": { "enable": true, "radio": 0, "if": -200000 },
+                "chan_multiSF_2": { "enable": true, "radio": 0, "if": 0 },
+                "chan_multiSF_3": { "enable": true, "radio": 0, "if": 200000 },
+                "chan_multiSF_4": { "enable": true, "radio": 1, "if": -300000 },
+                "chan_multiSF_5": { "enable": true, "radio": 1, "if": -100000 },
+                "chan_multiSF_6": { "enable": true, "radio": 1, "if": 100000 },
+                "chan_multiSF_7": { "enable": true, "radio": 1, "if": 300000 }
+              }
+            ],
+            "nocca": true,
+            "nodc": true,
+            "nodwell": true
+          }
+        }
+        ```
+
+      </details>
+
+   4. A more thorough description of `sx1301_conf` can be found at [The LNS Protocol](https://doc.sm.tc/station/tcproto.html?highlight=sx1301conf#router-config-message) specification.
+
+3. If you want to enable client certificate validation for this device, make sure to define the `properties.desired.clientThumbprint` desired property as an array of strings (each of them being one of the allowed thumbprints for client certificates of this device)
+
+4. If you want to enable CUPS for this device, you will need to:
+
+    1. upload the .bundle credential file to the 'stationcredentials' container in the Azure Function Storage Account created by the Starter Kit template ([Quickstart: Upload, download, and list blobs - Azure portal - Azure Storage | Microsoft Docs](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#upload-a-block-blob))
```

Comment:
where's the explanation on this file? The ADR mentions some things, but it maybe not be completely clear.

Reply:
I am adding explanation and link. Please resolve if ok